app/models/alpaca.rb

```diff
@@ -18,6 +18,13 @@ class Alpaca < ApplicationRecord
 
   has_many :ownerships
   has_many :users, through: :ownerships
+  
+  def last_owner
+    if ownerships.last != nil
```

Comment:
if !ownerships.last.nil?

Reply:
It works. Then nothing will happen. I can add description that Alpaca has no owner. But in the final version there will be no  Alpacas without owners.